ideas

```diff
@@ -73,13 +75,17 @@ Portals
         - Dynamically open wormhole as you approach the portal
         - Remote end of the portal is visible through the portal 
         
-        
 ---------------------
 
 Terrain
 
     - Many types of terrain -- dungeons, caves, forest, water, etc
     - would be cool to have large ground level + multiple entrances to underground
+ 
+
+** as imagination heavy as this is e.g. cursor and char monsters, we should consider how much time we really want to spend on graphics like
+forest and water. If we're adding forest, water, portals, light-up spells, etc, maybe we should just be building a full graphical game rather than spending the time
+building the ascii representation.
```

Comment:
1) This game is totally retro.
2) Art takes a lot of work. Let's focus on game mechanics / content, and consider adding more detail later
3) I think it would be wicked awesome if the art quality changes like ascii -> atari -> nes -> snes in mid-game. Maybe the deeper you go into the dungeon, or by activating some magical artifact, etc.